mergify_engine/queue.py

```diff
@@ -37,42 +37,34 @@ class QueueConfig(typing.TypedDict):
 
 @dataclasses.dataclass
 class Queue:
-    redis: utils.RedisCache
-    owner_id: int
-    owner: str
-    repo_id: int
-    repo: str
+    repository: context.Repository
     ref: str
 
     log: logging.LoggerAdapter = dataclasses.field(init=False)
 
     def __post_init__(self):
         self.log = daiquiri.getLogger(
```

Comment:
It'd be great to have a method in utils that would give a logger like this:
```python
def get_logger(name: str, gh_owner: github_types.GitHubLogin, gh_repo: github_types.GitHubRepositoryName, …):
   return daiquiri.getLogger(name, gh_owner=gh_owner, gh_repo=gh_repo, …)
```

So we're sure to always have logger with the same format and the type being checked.

Reply:
Agreed, maybe another time, because I have other issues in mind with all these loggers.